cmd/root.go

```diff
@@ -54,7 +54,7 @@ func initConfig(c *cobra.Command, args []string) error {
 		// Use config file from the flag.
 		viper.SetConfigFile(configFile)
 	} else {
-		viper.SetConfigName(".klepto")
+		viper.SetConfigName(".klepto.toml")
```

Comment:
This is not necessary. You can have a configuration file in any format (toml, yaml, json), viper should handle this for us.

Reply:
Ah, makes sense. Changed config name in the command example and `init()` function.